open-hackathon-server/src/hackathon/hack/hackathon_manager.py

```diff
@@ -41,6 +41,9 @@
     FILE_TYPE, HACK_TYPE, HACKATHON_STAT
 from hackathon import RequiredFeature, Component, Context
 
+import lxml
+from lxml.html.clean import Cleaner
```

Comment:
Please move those imports above imports from hackathon. As a conversion, we want to organize imports in three parts separated by a blank line. 
- modules/class/objects imported from python itself
- those from third-party libraries which have to install via `pip` 
- those from OHP